weaviate/connect/connection.py

```diff
@@ -458,6 +470,36 @@ def timeout_config(self, timeout_config: Union[Tuple[Real, Real], Real]):
     def proxies(self) -> dict:
         return self._proxies
 
+    def wait_for_weaviate(self, wait_for_weaviate: Optional[int]):
+        """
+        Waits until weaviate is ready or the timelimit given in 'wait_for_weaviate' has passed.
+
+        Parameters
+        ----------
+        wait_for_weaviate : Optional[int]
+            Describes how long the client will wait for weaviate to start in seconds.
+
+        Raises
+        ------
+        ConnectionError
+            If weaviate takes longer than the timelimit to respond.
+        """
+
+        ready_url = self.url + self._api_version_path + "/.well-known/ready"
+        for _i in range(0, wait_for_weaviate, 1):
```

Comment:
The step of `1` in the `range` function is the default one, we can remove it so it look cleaner.